Automattic/wp-calypso
# AT: Add new eligibility hold for unverified email address

Adds support for showing a new eligibility hold for an unverified email address.

**To test**
* Install a plugin or theme with an account that has not verified the email address. The hold should show and not allow you to proceed. 
* Verify the email address and return to the plugin or theme installation. You should be allowed to continue.

<img width="731" alt="screen shot 2017-03-16 at 3 21 03 pm" src="https://cloud.githubusercontent.com/assets/789137/24021076/4a42dd1e-0a5c-11e7-8c6d-6fc501caa840.png">

cc @rralian @drw158 @Copons @vindl
